### DISCUSSION TITLE
How to use rix from R provided by nix?

### DISCUSSION THREAD

**Discussion by frafra** [ropensci/rix#282] [Q&A] [answered]
rix is not an available package, and it is not possible to install R packages within R when provided by nix. Isn't that a bit ironic? :)

**Comment by b-rodrigues** [comment, marked as answer]
rix isn’t on CRAN yet, so it’s not on nixpkgs either. Once it’s on CRAN, it’ll eventually land on nixpkg which will make it easier to add to a Nix environment. 
If you want to include rix in a Nix environment, you should add it as a Github package as described here: https://b-rodrigues.github.io/rix/articles/d1-installing-r-packages-in-a-nix-environment.html

rix comes with a default.nix that includes rix as well, that you can take inspiration from: https://github.com/b-rodrigues/rix/blob/master/inst/extdata/default.nix

**Comment by b-rodrigues** [comment]
rix is on CRAN now and also available in nixpkgs now :)
